content/docs/use-cases/shared-development-server.md

```diff
@@ -62,7 +62,8 @@ files:
 $ dvc config cache.shared group
 ```
 
-> See `dvc config cache` for more information on these config options.
+> See `dvc config cache` and `dvc cache dir` for more information on these
+> config options.
```

Comment:
Good catch. Let's please invert the order (cache dir link before config cache) 🙂